packages/theme/src/Components/SearchableList/SearchableList.tsx

```diff
@@ -72,7 +73,7 @@ export function SearchableList<T>({
                 shouldSort: true,
                 threshold: 0.45,
                 minMatchCharLength: 1,
-                keys: searchKey ?? Object.keys(data.length > 0 ? data[0] : []),
+                keys: searchKey ?? Object.keys(data.length > 0 ? data[0] : EMPTY_LIST),
```

Comment:
No need to use EMP_LIST here
```suggestion
                keys: searchKey ?? data.length > 0 ? Object.keys(data[0]) : [],
```